platform/darwin/test/MGLStyleTests.mm

```diff
@@ -140,7 +132,7 @@ - (void)testStyleURLDeclarations {
     NSString *styleHeader = self.stringWithContentsOfStyleHeader;
 
     NSError *versionedMethodError;
-    NSString *versionedMethodExpressionString = @(R"RE(^\+\s*\(NSURL\s*\*\s*\)\s*\w+StyleURLWithVersion\s*:\s*\(\s*NSInteger\s*\)\s*version\s*;)RE");
+    NSString *versionedMethodExpressionString = @(R"RE(^\+\s*\(NSURL\s*\*\s*\)\s*\w+StyleURLWithVersion\s*:\s*\(\s*NSInteger\s*\)\s*version\s*)RE");
```

Comment:
Might be a good idea to add a `\b` where the `;` was, to ensure that all the methods have the same parameter name (and not something like `versions`).